docs/manual-CN/deploy-cluster-on-docker.md

```diff
@@ -0,0 +1,243 @@
+
+storaged 也使用 raft 协议保证高可用，在数据迁移时会与 metad 通信，所以需要配置 metad 的地址和端口 `meta_server_addrs` 和本机地址 `local_ip` ，其 peers 可以通过 metad 获得。cluster-2 上的部分配置选项如下：
+
+```
+# Meta server address
+--meta_server_addrs=172.17.0.2:45500,172.17.0.3:45500,172.17.0.4:45500
+# Local ip
+--local_ip=172.17.0.2
+# Storage daemon listening port
+--port=44500
+```
+
+请重复上述步骤，为 cluster-3、cluster-4 进行配置，一共需配置9个文件。
+
+
+## 启动集群
+
+进入容器后，默认在根目录下 / ，切换到 Nebula 主目录:
+
+```bash
+$ cd /usr/local/nebula/
+```
+
+集群配置完成后需重启服务：
+
+```bash
+$ /usr/local/nebula/scripts/nebula.service stop all
+[INFO] Stopping nebula-metad...
+[INFO] Done
+[INFO] Stopping nebula-graphd...
+[INFO] Done
+[INFO] Stopping nebula-storaged...
+[INFO] Done
+
+$ /usr/local/nebula/scripts/nebula.service start all
+[INFO] Starting nebula-metad...
+[INFO] Done
+[INFO] Starting nebula-graphd...
+[INFO] Done
+[INFO] Starting nebula-storaged...
+[INFO] Done
+```
+
+重复以上命令重启 cluster-3、cluster-4。
+
+
+## 测试集群
+
+登录集群中的一台，执行如下命令：
+
+```bash
+$ /usr/local/nebula/bin/nebula -u user -p password --addr 172.17.0.2 --port 3699
+
+Welcome to Nebula Graph (Version 5f656b5)
+
+(user@172.17.0.2) [(none)]> show hosts
+=============================================================================================
+| Ip         | Port  | Status | Leader count | Leader distribution | Partition distribution |
+=============================================================================================
+| 127.17.0.2 | 44500 | online | 0            |                     |                        |
+---------------------------------------------------------------------------------------------
+| 172.17.0.3 | 44500 | online | 0            |                     |                        |
+---------------------------------------------------------------------------------------------
+| 172.17.0.4 | 44500 | online | 0            |                     |                        |
+---------------------------------------------------------------------------------------------
+```
+
+表明三台集群已部署成功，插入[数据](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-CN/get-started.md#%E5%88%9B%E5%BB%BA%E5%9B%BE%E6%95%B0%E6%8D%AE)进行测试。
+
+```
+$a=GO FROM 201 OVER like yield like._dst as id; GO FROM $a.id OVER select YIELD $^.student.name AS Student, $$.course.name AS Course, select.grade AS Grade
+=============================
+| Student | Course  | Grade |
+=============================
+| Monica  | Math    | 5     |
+-----------------------------
+| Monica  | English | 3     |
+-----------------------------
+| Jane    | English | 3     |
+-----------------------------
+```
+
+停止 cluster-4 的 storage 进程：
+
+```bash
+$ /usr/local/nebula/scripts/nebula.service stop storaged
+```
+
+查看进程状态，确认进程是否停止：
+
+```bash
+$ /usr/local/nebula/scripts/nebula.service status storaged
+[INFO] nebula-storaged: Exited
+```
+
+登录 cluster-2，使用 `SHOW HOSTS` 查看：
+
+```
+> SHOW HOSTS
+==============================================================================================
```

Comment:
show hosts has been updated, it will show space name instead of space Id in `Leader distribution` and `Partition distribution`.